Dev/VideoChatClient/Input/IInput.hpp

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#include <cstdint>
+#include <vector>
+
+typedef std::vector<uint8_t> Buffer;
```

Comment:
Create a Common.hpp file and put this there, change in all files.

Reply:
How would I create it in both branches simultaneously?